### ISSUE TITLE
🐞 Bug report: Change color checkbox acceptance in forms

### ISSUE THREAD

**Issue by laurapnunez** [elementor/elementor#18222]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

If impossible to change color checkbox acceptance.

When a user click in checkbox acceptance it show a icon with a blue color that it is impossible change

### Steps to reproduce

A new form,

add item acceptance

When you go to the form and the user click in checkbox. It show a icon that is not possible change this icon and this colour

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.3.34
	PHP Version: 7.4.28
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.2
	Site URL: https://runki.org
	Home URL: https://runki.org
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: es
	Timezone: 0
	Admin Email: info@nasassocialmedia.com
	Debug Mode: Inactive

== Theme ==
	Name: Kadence
	Version: 1.1.15
	Author: Kadence WP
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: es_ES
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.74 Safari/537.36

== Active Plugins ==
	Accessibility Lite - A super lightweight Wordpress plugin
		Version: 3.6.1
		Author: Sitelinx

	Countdown Timer for WordPress Block Editor
		Version: 1.0.5
		Author: Flickdevs

	Draw Attention
		Version: 1.9.30
		Author: N Squared

	Elementor
		Version: 3.5.6
		Author: Elementor.com

	Elementor Pro
		Version: 3.6.4
		Author: Elementor.com

	GA Google Analytics
		Version: 20220124
		Author: Jeff Starr

	GDPR Cookie Consent
		Version: 2.1.1
		Author: WebToffee

	Kadence Pro - Premium addon for the Kadence Theme
		Version: 1.0.5
		Author: Kadence WP

	Limit Login Attempts Reloaded
		Version: 2.23.2
		Author: Limit Login Attempts Reloaded

	One Click Accessibility
		Version: 2.0.3
		Author: One Click Accessibility

	Premium Addons for Elementor
		Version: 4.9.2
		Author: Leap13

== Experimentos de Elementor ==
	Salida del DOM optimizada: Activo por defecto
	Carga de recursos mejorada: Activo por defecto
	Carga de CSS mejorada: Activo por defecto
	Fuentes de iconos integradas: Inactivo por defecto
	Mejoras de accesibilidad: Activo por defecto
	Importar o exportar un kit de plantillas: Activo por defecto
	Puntos de ruptura personalizados adicionales: Activo por defecto
	Ocultar los widgets nativos de WordPress de los resultados de búsqueda: Activo por defecto
	Por defecto, el nuevo constructor de temas: Activo por defecto
	Páginas de destino: Activo por defecto
	Muestrario de colores: Activo por defecto
	Widgets favoritos: Activo por defecto
	Barra superior de administración: Activo por defecto
	Page Transitions: Activo por defecto
	Envíos de formulario: Activo por defecto
	Ajuste de scroll: Activo por defecto

== Registro ==
	
JS: showing 2 of 2JS: 2022-02-22 16:40:00 [error X 3][https://runki.org/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.5.5:2:453832] Cannot read properties of null (reading \'getBoundingClientRect\') 
JS: 2022-03-17 09:22:18 [error X 11][https://runki.org/wp-content/plugins/elementor-pro/assets/js/frontend.min.js?ver=3.6.3:2:10003] Failed to execute \'observe\' on \'IntersectionObserver\': parameter 1 is not of type \'Element\'. 

Log: showing 20 of 202022-02-22 16:53:40 [info] elementor-pro::elementor_pro_updater Started 
2022-02-22 16:53:40 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-02-22 16:53:40 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-02-22 16:53:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.2',
  'to' => '3.6.2',
)]
2022-02-22 16:53:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.2',
  'to' => '3.6.2',
)]
2022-03-11 08:30:50 [info] elementor::elementor_updater Started 
2022-03-11 08:30:50 [info] Elementor/Upgrades - _on_each_version Start  
2022-03-11 08:30:50 [info] Elementor/Upgrades - _on_each_version Finished 
2022-03-11 08:30:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.5',
  'to' => '3.5.6',
)]
2022-03-11 08:30:50 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.5',
  'to' => '3.5.6',
)]
2022-03-14 15:33:54 [info] elementor-pro::elementor_pro_updater Started 
2022-03-14 15:33:54 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-03-14 15:33:54 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-03-14 15:33:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.2',
  'to' => '3.6.3',
)]
2022-03-14 15:33:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.2',
  'to' => '3.6.3',
)]
2022-03-17 11:16:20 [info] elementor-pro::elementor_pro_updater Started 
2022-03-17 11:16:20 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-03-17 11:16:20 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-03-17 11:16:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.3',
  'to' => '3.6.4',
)]
2022-03-17 11:16:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.3',
  'to' => '3.6.4',
)]

PHP: showing 3 of 3PHP: 2022-02-22 17:04:28 [notice X 2][/home/coqcnlfz/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-02-22 17:11:04 [notice X 1][/home/coqcnlfz/public_html/wp-content/plugins/elementor/includes/managers/image.php::87] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-03-15 17:21:55 [error X 2][/home/coqcnlfz/public_html/wp-content/plugins/elementor/core/documents-manager.php::533] Uncaught Error: Call to a member function get_post() on bool in /home/coqcnlfz/public_html/wp-content/plugins/elementor/core/documents-manager.php:533
Stack trace:
#0 [internal function]: Elementor\Core\Documents_Manager->ajax_save(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /home/coqcnlfz/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/coqcnlfz/public_html/wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /home/coqcnlfz/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)
#4 /home/coqcnlfz/public_html/wp-includes/plugin.php(474): WP_Hook->do_action(Array)
#5 /home/coqcnlfz/public_html/wp-admin/admin-ajax.php(187): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Premium Addons for Elementor: Compatible

== Elementor Pro - Compatibility Tag ==
	
	Premium Addons for Elementor: Compatible

**Comment by rseitzberg** [comment]
I think this is related. It is impossible to change the color of the checkbox checkmark, the border color, the background color or the label of the field in an Elementor form.  You have to use Custom CSS. 

I think I have seen third party plugins that add consent forms that you can style separately, but elementor support told me that the checkmark, background, and border can not, and the label font can not be manipulated either from within Elementor. It requires custom CSS. 

I was able to change it with custom CSS but this seems like an essential function for a consent form to change the font size or color independent of the other labels in the form. The checkmark too.

Thanks
